src/Microsoft.Sbom.Api/Executors/ComponentDetectionBaseWalker.cs

```diff
@@ -47,6 +47,14 @@ public abstract class ComponentDetectionBaseWalker
         {
             log.Debug($"Scanning for packages under the root path {buildComponentDirPath}.");
 
+            // If the buildComponentDirPath is null or empty, make sure we have a ManifestDirPath create a new temp directory with a random name.
+            if (!Directory.Exists(configuration.BuildComponentPath?.Value) && Directory.Exists(configuration.ManifestDirPath?.Value))
+            {
+                log.Debug($"The build component directory path {buildComponentDirPath} does not exist. Creating a new temp directory.");
```

Comment:
Since this is an implementation detail, I wouldn't log it

Reply:
I considered changing the message to something about scanning a docker image but this would cause it to logged along with other workflows that occur when we scan the empty temp path we create. So in the end I opted for just removing it. Let me know if you have any other suggestions.